File: Sep/Day8/convert-integer-to-the-sum-of-two-no-zero-integers.java
//  Time: O(N * LogN)  ::  Space: O(1)
class Solution {
    public int[] getNoZeroIntegers(int n) {
        int a = 1, b = n - a;
        while ((String.valueOf(a).contains("0") || String.valueOf(b).contains("0")) && a <= b) {
            a++;
            b--;
        }
        return new int[]{a, b};
    }
}

File: Sep/Day8/convert-integer-to-the-sum-of-two-no-zero-integers.py
#  Time: O(N * LogN)  ::  Space: O(1)
class Solution:
    def getNoZeroIntegers(self, n: int) -> List[int]:
        a = 1
        b = n-a
        while '0' in str(a) + str (b) and a <= b:
            a+=1
            b-=1
        return [a,b]

        